code/modules/power/solar.dm

```diff
@@ -209,7 +209,7 @@
 			user.visible_message("[user] wrenches the solar assembly into place.", "<span class='notice'>You wrench the solar assembly into place.</span>")
 			W.play_tool_sound(src, 75)
 		else
-			user.visible_message("[user] unwrenches the solar assembly from its place.", "<span class='notice'>You unwrench the solar assembly from its place.</span>")
+			user.visible_message("[user] unwrenched the solar assembly from its place.", "<span class='notice'>You unwrench the solar assembly from its place.</span>")
```

Comment:
Line 209: when wrenching, you are in the present
Line 212: when yanking, you are in the past